pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -815,28 +815,34 @@ cdef class _Timedelta(timedelta):
 
     cpdef timedelta to_pytimedelta(_Timedelta self):
         """
-        return an actual datetime.timedelta object
-        note: we lose nanosecond resolution if any
+        Return an actual datetime.timedelta object.
```

Comment:
can you make this an actual doc-string (e.g. have a Notes section)

Reply:
@jreback I ended up editting the whole doc-string...